templates/admin/login.html

```
@@ -0,0 +1,48 @@
+
+            {% if form.errors %}
+            <div class="login-error">
+                아이디 또는 비밀번호를 다시 확인해주세요.
+            </div>
+            {% endif %}
+
+            <form action="{{ app_path }}" method="post" id="login-form">
+                {% csrf_token %}
+
+                <div class="form-row">
+                    <input type="text" name="username" id="id_username" class="login-input" placeholder="아이디" autofocus
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)